WORKING_GROUPS.md

```diff
@@ -99,7 +99,7 @@ The i18n working groups handle more than just translations. They
 are endpoints for community members to collaborate with each
 other in their language of choice.
 
-Each team is organized around a common spoken language. Each
+Each team is organized around a commonly spoken language. Each
```

Comment:
IMHO commonly changes the very meaning of that sentence. Nothing here needs to imply that the language should be commonly spoken one. Only that the language should be common language of the team members.